src/adapters/mc/ssh/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# SO binary name must match the class name in MOF with a "lib" prefix added
+project(OsConfigResource)
```

Comment:
Can we reflect the project name to be equivalent to the target name `project(OsConfigResourceSsh)`

Reply:
Same as the other. Let's keep this for future improvement.